docs/resource-schemas.md

```diff
@@ -335,3 +367,36 @@ For the value of the `demoLinks` array, you can set the following keys.
 
 Libraries and assets have `id`s to uniquely identify each resource and establish relationships
 between resources. More guidance around format and absolute identifiers coming here soon!
+
+### Nav data
+
+Libraries and assets have documentation pages that are specific to their library. The `navData`
+array determines what MDX pages get rendered in the web app, as well as the link order in the nav.
+
+**Example**
+
+carbon.yml
+
+```yml
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@v1/carbon-resources.schema.json
+---
+navData:
+  - title: A page
+    path: '/a-page'
+  - title: Another page
+    items:
+      - title: Sub page
+        path: '/another-page/sub-page'
+        hidden: true
+      - title: Another sub page
+        path: '/another-page/another-sub-page'
+```
+
+For the value of the `navData` array, you can set the following keys.
+
+| Nav data | Description                                                  | Required | Type    | Default |
+| -------- | ------------------------------------------------------------ | -------- | ------- | ------- |
+| `title`  | Navigation title.                                            | Required | String  | –       |
+| `path`   | Relative path to the file.                                   | Required | String  | –       |
```

Comment:
Do we need to say that this is the relative path to the file - relative to this `carbon.yml`? How do we prevent people from using a path relative to the repo root?

Reply:
Updated to say `Path to the file, relative to carbon.yml.`